Objects/typeobject.c

```diff
@@ -1038,7 +1038,7 @@ type_modified_unlocked(PyTypeObject *type)
        We don't assign new version tags eagerly, but only as
        needed.
      */
-    if (type->tp_version_tag == 0) {
+    if (FT_ATOMIC_LOAD_UINT32_RELAXED(type->tp_version_tag) == 0) {
```

Comment:
`tp_version_tag` is declared as `unsigned int` so we should use `FT_ATOMIC_LOAD_UINT_RELAXED` (instead of `uint32`)